src/plugins/Hacktoberfest/hacktoberfest.ts

```diff
@@ -7,7 +7,7 @@ import { WebhookPayloadIssuesIssue } from "@octokit/webhooks";
 
 const NAME = "Hacktoberfest";
 
-const isHacktoberfestLive = () => new Date().getMonth() == 9;
+export const isHacktoberfestLive = () => new Date().getMonth() == 10;
```

Comment:
Isn't it zero index based:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth

Reply:
hmm... it seems it is...